pegasus/sites.v3/code.org/public/congrats.haml

```diff
@@ -26,7 +26,10 @@ max_age: 60
 -twitter[:hashtags] = 'HourOfCode' unless I18n.t(:just_did_hoc).include? '#HourOfCode'
 
 -company_param = params[:co]
--form = DB[:forms].where(kind:'CompanyProfile', name:company_param).first if company_param
+-# Our partnership with cartoon network piggy-backs on our company profile infrastructure.
```

Comment:
"Do not show company profile sections for the cartoon network, even if someone accidentally creates an entry for them in the forms table.  (Our partnership with cartoon network piggy-backs on our company profile infrastructure.)"